modules/ROOT/pages/to-deploy-using-rest-connect.adoc

```diff
@@ -1,28 +1,26 @@
-= Converting a RAML to a Connector Using REST Connect
+= REST Connect Connector Generator
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:keywords: faq, rest connect, connect, rest
+:imagesdir: ../assets/images/
 
-REST Connect quickly converts a RAML 1.0 API specification to a connector, which you can use in a Mule application in the Design Center feature of Anypoint Platform and in Anypoint Studio.
+The Exchange backend uses REST Connect to transparently convert a REST API specification to a Mule 3 and Mule 4 connector. You can use this connector as you would any other in Anypoint Studio.
 
-When a user clicks the Publish to Exchange button, a REST API asset is created in Anypoint Exchange.
+REST Connect supports RAML v1.0. If you publish a RAML v0.8 API Specification, you receive an alert email. To have an auto-generated connector, convert your API specification to RAML v1.0.
```

Comment:
I don't think that Rest Connect is sending emails when publishing RAML v0.8. @pmantellini @giordanolucas can you confirm?

Reply:
We are not sending emails for RAML 0.8 errors since some customers want to keep using RAML 0.8 without getting error emails every time they publish a new API.

Reply:
Removed that paragraph.